ocp_utilities/operators.py

```diff
@@ -358,17 +358,18 @@ def _icsp(_repository_digest_mirrors: List[Dict[str, Any]]) -> None:
 
     brew_registry = "brew.registry.redhat.io"
     source_iib_registry = iib_index_image.split("/")[0]
+    brew_image_repo = iib_index_image.split("/")[1]
     _iib_index_image = iib_index_image.replace(source_iib_registry, brew_registry)
     icsp = ImageContentSourcePolicy(name="brew-registry")
     validating_webhook_configuration = ValidatingWebhookConfiguration(name="sre-imagecontentpolicies-validation")
     repository_digest_mirrors: List[Dict[str, Any]] = [
         {
-            "source": source_iib_registry,
```

Comment:
Just a FYI, you may want to update the code to support idms. ICSP is deprecated: https://docs.openshift.com/container-platform/4.15/openshift_images/image-configuration.html 
```
Using an ImageContentSourcePolicy (ICSP) object to configure repository mirroring is a deprecated feature. Deprecated functionality is still included in OpenShift Container Platform and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. If you have existing YAML files that you used to create ImageContentSourcePolicy objects, you can use the oc adm migrate icsp command to convert those files to an ImageDigestMirrorSet YAML file. For more information, see "Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring" in the following section.
--


```
While there is no clear information when it would be removed, it would be good to move any deployment job and test code to use idms sooner.

Reply:
@dbasunag Thanks, but move to ImageDigestMirrorSet should be in different PR